config/features.rb

```
@@ -4,6 +4,7 @@
   strategy :split_client, api_key: ENV['SPLIT_API_KEY'] if ENV['SPLIT_API_KEY'].present?
 
   feature :foo, description: 'Example feature flag', default: true
+  feature :health_check, description: 'Dummy feature for split.io health check'
```

can we use foo instead?

---

`foo` is already defaulted to true in the features file, so returns true even if we can't read from split.io. I added this explicit feature with a default value of false, and is only enabled via split.io so we can be sure the API call is working.

---

My preference would be cascading on their own health check, if there is one. Relying on a a dummy feature status could render the whole service inoperative if we toggle that feature by mistake or if it defaults to the wrong thing in an split.io outage.

---

Split.io don't have anything like that, so this is as good as we can do really. It does exercise the split.io API - and if we did change the feature setting either on split.io (to off) then healthcheck would fail and we'd know. If someone changed the default in the features file the unit tests would fail and we'd know. So I think we're good here.